habitat/plan.sh

```diff
@@ -9,19 +9,19 @@ pkg_lib_dirs=(lib)
 pkg_svc_user=root
 pkg_svc_group=${pkg_svc_user}
 pkg_build_deps=(
-  core/bundler
   core/gcc-libs
   core/git
-  core/libffi
   core/make
   core/openssl
-  core/postgresql-client
   core/rsync
 )
 pkg_deps=(
   core/coreutils
   core/gcc
   core/ruby
+  core/bundler
```

Comment:
we should probably look into this a bit more, we might not need bundler if we give it the right flags

Reply:
there's a `--standalone` flag

Reply:
If we're using ruby 2.6+ it comes with bundler built in

Reply:
or hab package is 2.5.7 and seems to have it